lib/fika/expressions.ex

```diff
@@ -24,8 +24,15 @@ defmodule Fika.Parser.Expressions do
       non_literal_exps
     ])
 
+  exp_unary_op =
+    choice([string("+"), string("-")])
+    |> lookahead_not(choice([string("+"), string("-")]))
+    |> concat(allow_horizontal_space)
+    |> concat(factor)
+    |> Helper.to_ast(:unary_op)
+
   term =
-    factor
+    choice([exp_unary_op, factor])
```

Comment:
Can we add the `exp_unary_op` at the end of the non_literal_exps choices instead of adding the choice here in `term`?